base/setup/welcome/CMakeLists.txt

```diff
@@ -4,4 +4,24 @@ add_rc_deps(welcome.rc ${welcome_rc_deps})
 add_executable(welcome welcome.c welcome.rc)
 set_module_type(welcome win32gui UNICODE)
 add_importlibs(welcome gdi32 user32 shell32 msvcrt kernel32 ntdll)
+
+# Reduce the required subsystem to WinNT 4.0 for i386 builds only.
+if(ARCH STREQUAL "i386")
+    if(MSVC)
+        # add_target_link_flags(welcome "/SUBSYSTEM:WINDOWS,4.00")
+
+        # Retrieve the full path to the generated file of the 'welcome' target,
+        # then change the subsystem version
+        get_target_property(_welcome_exe_file welcome LOCATION)
+        add_custom_command(
+            TARGET welcome POST_BUILD
+            COMMAND editbin.exe /SUBSYSTEM:WINDOWS,4.00 ${_welcome_exe_file}
```

Comment:
Just make this `$<TARGET_FILE:welcome>` and you don't need the LOCATION hack.

Reply:
Yep, I've fixed that locally because your policy changes regarding this makes my cmake erroring out (strangely, no warning/error appears in the build bots).